JoinColony/colonyCDapp
# Fix: Disable click for pending transactions in Userhub

## Description

This just simply disables click for pending transactions in the UserHub

![disable pending click](https://github.com/user-attachments/assets/02d8048d-bacf-4cca-aab4-a858a77c5d2a)

## Testing

1. Make a Simple Payment
2. As soon as the Pending button comes up, click it
3. The UserHub should come up and and you should automatically land on the Transactions tab
4. Click your Transaction while it's in a Pending state
5. Verify that you are not able to
6. Verify that the click is successful only when the Transaction exits the Pending state

Resolves #2738
